tools/fvm-bench/src/main.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright 2021-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+mod fevm;
+
+use std::fs;
+
+use anyhow::{anyhow, Context};
+use clap::Parser;
+
+/// Run a contract invocation for benchmarking purposes
+#[derive(Parser, Debug)]
+struct Args {
+    /// Execution mode: wasm or fevm
+    #[arg(short, long, default_value = "fevm")]
+    mode: String,
+
+    /// Emit debug logs
+    #[arg(short, long, default_value = "false")]
+    debug: bool,
+
+    /// Emit detailed gas tracing information
+    #[arg(short, long, default_value = "false")]
+    trace: bool,
+
+    /// Emit user generated logs
+    #[arg(short, long, default_value = "false")]
+    events: bool,
+
+    /// Builtin actors bundle to use.
+    #[arg(short, long)]
+    bundle: String,
+
+    /// Contract file.
+    contract: String,
```

Comment:
Okay so what if the contract has a constructor with non-zero arguments? I just read that they should be appended to the `initcode`. If you expect developers to append those to the actual file, it would be worth mentioning it here. Or should the tool take further parameters?

Reply:
We can't just say constructor args are not supported whatsoever, when a Solidity contract can have constructor arguments.

Reply:
really?
We had this at the beginning and got removed aling the way.
Now it is just iniycode you pass.

cc @stebalien.

Reply:
I'm confused. What are you referring to here?

Reply:
Solidity constructors can have parameters, but parameters just get tacked onto the end of the initcode, so there probably doesn't need to be any additional code to support them.